src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/product/form/associations.js

```diff
@@ -396,6 +458,18 @@ define([
         this.renderGrid(currentGrid.name, currentGrid.getInitialParams(this.getCurrentAssociationType()));
         this.setListenerSelectors();
       }
+
+      isQuantifiedAssociation ?
+        this.$('#product-quantified-associations').removeClass('hide') :
+        this.$('#product-quantified-associations').addClass('hide')
+    },
+
+    isQuantifiedAssociation: function (associationTypes, associationTypeCode) {
+      const associationType =  associationTypes.find(associationType => associationType.code === associationTypeCode);
+
+      if (undefined === associationType) throw new Error(`Cannot find assocation type ${associationTypeCode}`);
+
+      return associationType.code === 'PACK';
```

Comment:
Now we have is_quantified  is returned by internal API we can update this 
```suggestion
      return associationType.is_quantified === true;
```